tests/support/util.tcl

```diff
@@ -74,6 +74,8 @@ proc sanitizer_errors_from_file {filename} {
 proc getInfoProperty {infostr property} {
     if {[regexp "\r\n$property:(.*?)\r\n" $infostr _ value]} {
         set _ $value
+    } elseif {[regexp "^$property:(.*?)\r\n" $infostr _ value]} {
+        set _ $value
     }
 }
```

Comment:
I found this to be a bit cleaner, it's referenced here: https://github.com/redis/redis/pull/10920/files. 
```suggestion
proc getInfoProperty {infostr property} {
    if {[regexp -lineanchor "^$property:(.*?)\r\n" $infostr _ value]} {
        return $value
    }
}
```